src/sage/graphs/graph.py

```diff
@@ -4901,16 +4901,23 @@ def independent_set_of_representatives(self, family, solver=None, verbose=0,
         return repr
 
     @doc_index("Algorithmically hard stuff")
-    def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3):
+    def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3, induced=False):
```

Comment:
You should put `induced=False` before the `*`. The `*` forces to specify the name of the parameter.